src/backend/cuda/CMakeLists.txt

```diff
@@ -10,6 +10,18 @@ include(select_compute_arch)
 
 dependency_check(CUDA_FOUND "CUDA not found.")
 
+# Remove cublas_device library which is no longer included with the cuda
+# toolkit. Fixes issues with older CMake versions
+if(DEFINED CUDA_cublas_device_LIBRARY AND NOT CUDA_cublas_device_LIBRARY)
+  list(REMOVE_ITEM CUDA_CUBLAS_LIBRARIES ${CUDA_cublas_device_LIBRARY})
+endif()
+
+# Remove cublas_device library which is no longer included with the cuda
```

Comment:
I merely moved these to the top so that those changes are used by all cuda_add_library calls consistently. Yeah, they look the same, I didn't realize that while moving. Must have been the result of a three-way merge perhaps.

I will remove it in a later cleanup commit.